src/Rails.elm

```diff
@@ -1,9 +1,9 @@
-module Rails exposing (Error, get, post, put, delete, decodeErrors, csrfToken, request)
+module Rails exposing (Error, get, post, put, delete, decodeErrors, decodeRawErrors, csrfToken, request)
 
 {-|
 
 ## Requests
-@docs Error, get, post, put, delete, decodeErrors, request
+@docs Error, get, post, put, delete, decodeErrors, decodeRawErrors, request
 
 ## Customizing
 @docs csrfToken
```

Comment:
💭  Should we add a deprecation notice here to make life slightly easier if at some point in the future we want to remove this as part of a major release?